plugin/src/main/cpp/export/pico_export_plugin.cpp

```diff
@@ -61,14 +71,14 @@ PicoEditorExportPlugin::PicoEditorExportPlugin() {
 			pico::MANIFEST_FALSE_VALUE,
 			false);
 
-	_face_tracking_option = _generate_export_option(
```

Comment:
Can you leave the `_face_tracking_option` here to keep the history clean and add the new hand tracking option below.